.github/workflows/release.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-node@v4
         with:
-          node-version: "20"
+          node-version-file: ".nvmrc"
```

Comment:
i switched to .nvmrc to be able to use nvm (also added instructions into CONTRIBUTING.md)